code/modules/paperwork/stamps.dm

```diff
@@ -1,7 +1,7 @@
 /obj/item/stamp
 	name = "\improper GRANTED rubber stamp"
 	desc = "A rubber stamp for stamping important documents."
-	icon = 'icons/obj/bureaucracy.dmi'
+	icon = 'nsv13/icons/obj/bureaucracy.dmi'
```

Comment:
This should be changed on the master at arms stamp itself only

Reply:
origami.dm uses the nsv version of bureaucracy.dmi either way. Are you sure you paperwork and origami to use different dmi files for the same stamp?

Reply:
Jesse what the fuck are you talking about

Reply:
This icon variable is only used by the stamp object itself it doesn't make any sense to change it for anything that isn't specifically what we need
So just add `icon = 'nsv13/icons/obj/bureacracy.dmi'` to /obj/item/stamp/maa instead of here